src/validation.cpp

```diff
@@ -4112,11 +4112,12 @@ void BlockManager::Unload() {
     m_block_index.clear();
 }
 
-bool static LoadBlockIndexDB(ChainstateManager& chainman, const CChainParams& chainparams) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
+bool CChainState::LoadBlockIndexDB(const CChainParams& chainparams) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
```

Comment:
In commit "validation: Move LoadBlockIndexDB to CChainState" (fa730f7a871c49072e2e899cefbc8275e5f67057)

It seems chainman variable wasn't really needed here, just the chainman.m_blockman member, so this might make more sense as a BlockManager method instead of CChainState method

Reply:
Unfortunately we need `setBlockIndexCandidates` which is a member off `CChainState`... If you think `LoadBlockIndexDB` makes more sense logically as a method of `BlockManager` rather than `CChainState` we can pass in the `setBlockIndexCandidates`. Let me know!

Reply:
re: https://github.com/bitcoin/bitcoin/pull/21055#discussion_r578720086

> Unfortunately we need `setBlockIndexCandidates` which is a member off `CChainState`... If you think `LoadBlockIndexDB` makes more sense logically as a method of `BlockManager` rather than `CChainState` we can pass in the `setBlockIndexCandidates`. Let me know!

Thanks, I didn't recognize the `setBlockIndexCandidates` access. Proposed change sounds reasonable, but I don't actually know too much about it and it would seem good to tackle as a followup not changing here.